### PR TITLE
test: add tests for getCldImageUrl

### LINKED ISSUE #146: add tests for getCldImageUrl
We use [vitest](https://vitest.dev/) for our testing suite, and the tests are located in [packages/svelte-cloudinary/tests](./packages/svelte-cloudinary/tests). We just need some simple tests that cover basic functionality for now

### PR DESCRIPTION
**Pull request by moonlander101**
# Description

Includes tests to the getCldImageUrl function to test basic functionality.

## Issue Ticket Number

Fixes #146 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/cloudinary-community/svelte-cloudinary/issues) ticket for this PR
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/cloudinary-community/svelte-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have run tests locally to ensure they all pass
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: packages/svelte-cloudinary/tests/GetCldImageUrl/GetCldImageUrl.test.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { describe, it, expect } from 'vitest';
+import { getCldImageUrl } from '../../src/helpers/getCldImageUrl';
```
Comment (ghostdevv) [inline packages/svelte-cloudinary/tests/GetCldImageUrl/GetCldImageUrl.test.ts:2]:
```suggestion
import { getCldImageUrl } from '$src/helpers/getCldImageUrl';
```

**Review by ghostdevv — CHANGES_REQUESTED** [review]

**Review by ghostdevv — APPROVED** [review]
LGTM, ty!

**Comment by ghostdevv** [comment]
@all-contributors add @moonlander101 for test

**Comment by colbyfayock** [comment]
@moonlander101  thanks for your contribution. this PR is eligible for free Cloudinary Hackatoberfest swag. Please send me an email at [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your name, GitHub username, and a link to the PR where I'll send further instructions.

Happy Hacktoberfest!

**Pull request merged**